### PR TITLE
[Merged by Bors] - chore: cleanup imports of Data/Rat/Cast/Defs

### PR DESCRIPTION
**Pull request by kim-em**
Removing further unnecessary imports on the path to major tactics.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Rat/Cast/Lemmas.lean [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+/-!
+# Some exiled lemmas about casting
+
+These lemmas have been removed from `Mathlib.Data.Rat.Cast.Defs`
+to avoiding needing to import `Mathlib.Algebra.Field.Basic` there.
+
+In fact, these lemmas don't appear to be used anywhere in Mathlib,
+so perhaps this file can simply be deleted.
```
Comment (hrmacbeth) [inline Mathlib/Data/Rat/Cast/Lemmas.lean:19]: I'd be happy with deleting this file, but I'll leave it to your call.

**Review by hrmacbeth — COMMENTED** [review]
bors d+

File: Mathlib/Data/Rat/Cast/Lemmas.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Data/Rat/Cast/Lemmas.lean:19]: I'd prefer to leave this for someone who knows these lemmas better to delete later.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**